workers/unity/Packages/com.improbable.gdk.core/Schema/common.schema

```diff
@@ -0,0 +1,40 @@
+package improbable.core;
+import "improbable/vector3.schema";
+
+type EmptyMessage {}
+
+type BoolMessage {
+    bool value = 1;
+}
+
+type IntMessage {
```

Comment:
How about renaming this to `Int32Message`?

Reply:
There's actually 8 different types of int available. I'm not sure that it adds much to make each primitive type it's own message unless we are using them a lot.
Empty is the message that is really useful here.